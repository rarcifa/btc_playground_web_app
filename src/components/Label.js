import * as React from 'react';
import { InputLabel } from '@material-ui/core';

function Label() {

    return (
        <InputLabel
            htmlFor="outlined-adornment-amount">
            Clipboard
        </InputLabel>
    );
}

export default Label;