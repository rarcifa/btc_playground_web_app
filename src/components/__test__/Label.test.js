import React from 'react';
import { shallow } from 'enzyme';
import { InputLabel, MuiThemeProvider } from '@material-ui/core';
import Label from './../Label';
import theme from './../../theme';

const wrapper = shallow(
  <MuiThemeProvider theme={theme}>
    <Label />
  </MuiThemeProvider>
);

describe('Label', () => {
  it('should render the Label Component correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});