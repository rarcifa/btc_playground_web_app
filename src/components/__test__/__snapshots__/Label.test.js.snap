// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label should render the Label Component correctly 1`] = `ShallowWrapper {}`;
